src/main/java/org/opensearch/performanceanalyzer/collectors/StatsCollector.java

```diff
@@ -82,24 +97,58 @@ public static StatsCollector instance() {
         return statsCollector;
     }
 
+    @Override
+    public void collectMetrics(long startTime) {
+        Map<String, AtomicInteger> currentCounters = counters;
+        counters = new ConcurrentHashMap<>();
+
+        // currentCounters.putIfAbsent(StatExceptionCode.TOTAL_ERROR.toString(), new
+        // AtomicInteger(0));
```

Comment:
Do we need this? If not can we remove it?

Reply:
Done.